apps/assistants/forms.py

```diff
@@ -31,11 +49,22 @@ def __init__(self, request, *args, **kwargs):
             "x-model.number.fill": "llmProvider",
         }
         self.fields["llm_model"].widget.template_name = "django/forms/widgets/select_dynamic.html"
+        self.fields["include_file_info"].help_text = """If checked, extra information about uploaded files will
+            be appended to the instructions. This will give the assistant knowledge about the file types."""
```

Comment:
I'm a bit confused by these diffs from https://github.com/dimagi/open-chat-studio/pull/618 but I assume it's just an artifact.

Reply:
Sorry yes. My brain didn't register these changes are from a previously merged PR. I tested locally and all seems to be working fine.

Reply:
@snopoke can this PR be merged?